applications/GeoMechanicsApplication/tests/test_mohr_coulomb_with_tension_cutoff/test_dirichlet_corner_return_zone/ProjectParameters.json

```
@@ -0,0 +1,167 @@
+{
+    "problem_data": {
+        "problem_name":         "test_dirichlet_corner_return_zone",
+        "start_time":           0.0,
+        "end_time":             1.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "mesh"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "../common/MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":                1.0,
+            "max_delta_time_factor":    1
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  true,
+        "block_builder":                      true,
+        "solution_type":                      "Quasi-Static",
+        "scheme_type":                        "Backward_Euler",
+        "reset_displacements":                true,
+        "newmark_beta":                       0.25,
+        "newmark_gamma":                      0.5,
+        "newmark_theta":                      0.5,
+        "rayleigh_m":                         0.0,
+        "rayleigh_k":                         0.0,
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "residual_criterion",
+        "residual_relative_tolerance":        1.0E-4,
+        "residual_absolute_tolerance":        1.0E-9,
+        "min_iterations":                     6,
+        "max_iterations":                     15,
+        "number_cycles":                      10,
+        "reduction_factor":                   0.5,
+        "increase_factor":                    2.0,
```

We don't intend to use adaptive time stepping here, so we'd better set these values to 1.0 to communicate that:
```suggestion
        "reduction_factor":                   1.0,
        "increase_factor":                    1.0,
```